packages/ember-data/lib/system/relationships/belongs-to.js

```diff
@@ -92,6 +92,10 @@ function belongsTo(modelName, options) {
 
   opts = opts || {};
 
+  if (typeof opts.async === 'undefined') {
+    Ember.deprecate('In the future, relationships will be asynchronous by default. You must set { async: false } if you wish for a relationship remain synchronous.');
```

Comment:
Can you add which relationship to the deprecation notice please. This way it will be much more annoying to track down.